src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1554901333.inc.php

```diff
@@ -0,0 +1,10 @@
+<h1>Build #1551195244</h1>
+<h2>Date: 2019-02-26</h2>
+<div class="changelog">
+</div>
+<?php
+
+TCMSLogChange::addInfoMessage('Pages are now only available if they are part of a navigation (see navigation
+options in the portal settings). This restores the behavior of a previous Chameleon release; it was never intended that
+any pages are accessible in the frontend. Check if all pages that should be available in the frontend are part of a
```

Comment:
"any"? not "all"?

any would mean "more than zero" here.

Reply:
Improved wording.